ci/nightly/pipeline.template.yml

```diff
@@ -75,7 +75,7 @@ steps:
     steps:
     - id: feature-benchmark
       label: "Feature benchmark against merge base or 'latest' %N"
-      timeout_in_minutes: 720
+      timeout_in_minutes: 800
```

Comment:
With the parallelism it still runs this slow?!

Reply:
I though I have to increase it if I add scenarios.
Does this timeout apply to each parallel shard or do the shards get their fraction? I assumed the latter.

Reply:
Oh wow 🤯 
Reverting this change then.